ISPP-G5/NexONG_Frontend
# Feature/165-Recurring donations

- Fixed the partner regsiter to match the latest backend version.
- Changed the placeholder message to a request of account confirmation.

To test this feature:
1. Go to [http://localhost:3000/donaciones](http://localhost:3000/donaciones).
2. Create a new partner account in the right half of the screen.
3. If everithing goes alright, a request for confirmation will be shown.
4. Check your email for a message from [info.nexong@gmail.com](info.nexong@gmail.com).
5. Click the link in the message and click the "POST" button in the page you will be redirected to.
6. Go to [http://localhost:3000/iniciar-sesion](http://localhost:3000/iniciar-sesion) and log in with the user you just created. You should be sent to the partners section of the app.
